extras/module-node/duk_module_node.c

```diff
@@ -191,7 +191,7 @@ static duk_int_t duk__eval_module_source (duk_context *ctx, void *udata) {
 static duk_int_t duk__eval_module_source (duk_context *ctx) {
 #endif
 	/*
-	 *  Stack: [ module source ]
+	 *  Stack: [ ... module source ]
```

Comment:
I always forget safe calls don't get their own stack frame (yet).  Thanks for fixing this.